src/__stories__/list-story.tsx

```diff
@@ -103,6 +106,20 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
                     ),
                 };
                 break;
+            case 'custom element with text':
```

Comment:
Nice, I only miss an update at line 80 (see https://github.com/Telefonica/mistica-web/pull/407/files#diff-b0914293aa384640f5a0b98c3be2713a6dbd19d63fdcdc8a8723cb3eab4d931d), that uses `right` and should be `renderRight: null`. Also the comment.